task_12/index.html

```diff
@@ -0,0 +1,115 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge"/>
+    <title>Title</title>
+    <link href="https://fonts.googleapis.com/css?family=Ubuntu:300,300i,400,400i,500,500i,700,700i" rel="stylesheet">
+    <link rel="stylesheet" href="dist/css/style.css">
+    <script src="https://use.fontawesome.com/4a582f4e95.js"></script>
+    <script src="src/js/time.js"></script>
```

Comment:
Prefer to import scripts in the bottom of `body`. When scripts are loading parsing of HTML stops because script could contain clause, e.g. `document.write('<!--')`, which may change markup in all.

Reply:
I know about this since in this project js is only formally, I carried it up.
In the future, I want to study MVC, use the asymmetry and correctly solve this problem